package.json

```diff
@@ -110,19 +110,22 @@
     "@types/shelljs": "^0.8.5",
     "@types/styled-components": "^5.0.1",
     "autoprefixer": "^9.7.4",
+    "babel-jest": "^26.6.3",
```

Comment:
Hmm.. I think we need to wait until this #1003 PR has been merged first since current jest version is still v25 instead of v26 :bow:

Reply:
Do you know if anyone is still maintaining this repo? My mentioned PR is waiting for review for half a year.